src/app/containers/Article/index.test.jsx

```diff
@@ -6,8 +6,89 @@ import ArticleContainer from './index';
 global.console.log = jest.fn();
 
 describe('ArticleContainer', () => {
+  const articleData = {
+    metadata: {
+      id: 'urn:bbc:ares::article:c0000000001o',
+      locators: {
+        optimoUrn: 'urn:bbc:optimo:asset:c0000000001o',
+      },
+      type: 'article',
+      createdBy: '',
+      blockId: 'a-1',
+      created: 1514808060000,
+      firstPublished: 1514808060000,
+      lastPublished: 1514811600000,
+      lastUpdated: 1514811600000,
+      passport: {
+        language: 'en-gb',
+        home: 'http://www.bbc.co.uk/ontologies/passport/home/News',
+        articleType: 'news',
+        genre: null,
+      },
+    },
+    content: {
+      model: {
+        blocks: [
+          {
+            blockId: 'h-1',
+            type: 'headline',
+            model: {
+              blocks: [
+                {
+                  blockId: 't-1',
+                  type: 'text',
+                  model: {
+                    blocks: [
+                      {
+                        type: 'paragraph',
+                        blockId: 'p-1',
+                        model: {
+                          text: 'Article Headline',
+                        },
+                      },
+                    ],
+                  },
+                },
+              ],
+            },
+          },
+          {
```

Comment:
I wanted to use `textBlock` from `'models/blocks.js'` here, but that has an extra `{blocks: [..]}` surrounding it.  I could refactor `textBlock` to not have the surrounding array, but this would mean updating a several other components.